rpa-mapper/src/main/java/uk/gov/hmcts/cmc/rpa/mapper/RequestForJudgementJsonMapper.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.cmc.rpa.mapper;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.CountyCourtJudgment;
+import uk.gov.hmcts.cmc.domain.models.RepaymentPlan;
+import uk.gov.hmcts.cmc.rpa.DateFormatter;
+import uk.gov.hmcts.cmc.rpa.mapper.json.NullAwareJsonObjectBuilder;
+
+import java.util.Optional;
+import javax.json.JsonObject;
+import javax.json.JsonObjectBuilder;
+
+@Component
+public class RequestForJudgementJsonMapper {
+
+    public JsonObject map(Claim claim) {
+        CountyCourtJudgment countyCourtJudgment = claim.getCountyCourtJudgment();
+        return new NullAwareJsonObjectBuilder()
+            .add("caseNumber", claim.getReferenceNumber())
+            .add("ccjRequestedOn", DateFormatter.format(claim.getCountyCourtJudgmentRequestedAt()))
+            .add("amountWithInterest", claim.getTotalAmountTillToday().orElse(null))
+            .add("courtFee", claim.getClaimData().getFeesPaidInPound())
+            .add("alreadyPaid", countyCourtJudgment.getPaidAmount().orElse(null))
+            .add("paymentType", countyCourtJudgment.getPaymentOption().name())
+            .add("fullPaymentDeadline", countyCourtJudgment.getPayBySetDate().map(DateFormatter::format).orElse(null))
+            .add("instalments", getInstalments(countyCourtJudgment.getRepaymentPlan()))
```

Comment:
I would have preferred `countyCourtJudgment.getRepaymentPlan().map(this::toJSON()).orNull`. In that case we would have `"instalments": null` incase there is no repayment plan.

Reply:
Implemented it but nullable/empty structure is still needed in case replayment plan is not opted hence added an emptyJson method to get null valued structure.